common-travel.lic

```diff
@@ -180,4 +180,24 @@ module DRCT
     end
     return reverse_path
   end
+
+  def get_hometown_target_id(hometown, target)
+    hometown_data = get_data('town')[hometown]
+    target_id = hometown_data[target] && hometown_data[target]['id']
+    #try twice with pause if failed the first time, because sometimes data doesn't get loaded correctly
```

Comment:
❓ I've never seen data fail to load. Can you be more specific with what's going on?

Reply:
For whatever reason, every so often the roomid data for binning in steal.lic doesn't get loaded:
```
> ,steal
--- Lich: steal active.
--- Lich: jail-buddy active.
[steal: *** Binning is not yet supported in Shard]
```
Binning is definetly supported in Shard.  I also get the same thing for Crossing as well.

The only thing that triggers that message (in steal) is when you're set to bin, and the below code fails. 

https://github.com/rpherbig/dr-scripts/blob/f225d0fbaf81d903ee2ad71f56831fd7064c0718/steal.lic#L24-L30

Others have reported the same intermittently in lnet, to which advice has generally been, if you notice it, kill it restart, and it'll pick up.  I added a 2nd check to my steal.lic and never had problems past the 2nd load attempt (never got around to doing a PR).

I ran into the same issue when introducing this generic function, unless I tried a second time if it failed the first.